insights/specs/default.py

```diff
@@ -972,6 +972,7 @@ class DefaultSpecs(Specs):
     vsftpd_conf = simple_file("/etc/vsftpd/vsftpd.conf")
     watchdog_logs = glob_file("/var/log/watchdog/*.std*")
     wc_proc_1_mountinfo = simple_command("/usr/bin/wc -l /proc/1/mountinfo")
+    wc_var_lib_pcp_config_pmda = simple_command("/usr/bin/wc -l /var/lib/pcp/config/pmda/*")
```

Comment:
In the new `wc_files` specs, I suggest removing the `-l` option, in case some future specs may need to check the size of `chars` or `bytes`.